### ISSUE TITLE
Scorecard plugin: Remove accessibility violation filter and fix remaining a11y issues

### ISSUE THREAD

**Issue by christoph-jerolimov** [redhat-developer/rhdh-plugins#3590]
The scorecard insights plugins has playwright tests that include a bunch of accessibility assertions.

The tests runs `runAccessibilityTests` from `@workspaces/scorecard/packages/app-legacy/e2e-tests/utils/accessibility.ts`, but some violations was ignored at the moment. They are filtered out.

1. Remove the accessibility violation filter to fail if there is any a11y issue.
2. Run the playwright tests via `CI=1 yarn test:e2e` and list the left over issues.
3. Try to resolve these issues in the code base. When unsure, or not all issues can be removed, create a PR anyway.
